constants/repo.go

```diff
@@ -16,3 +16,21 @@ const (
 	// in Vela to control their pipeline being compiled as Starlark templates.
 	PipelineTypeStarlark = "starlark"
 )
+
+// Repo ApproveForkBuild types.
+const (
+	// ApproveAlways defines the CI strategy of having a repo administrator approve
+	// all builds triggered from a forked PR.
+	ApproveAlways = "always"
+
+	// ApproveNoWrite defines the CI strategy of having a repo administrator approve
+	// all builds triggered from a forked PR where the author does not have write access.
+	ApproveNoWrite = "no-write"
+
+	// ApproveOnce defines the CI strategy of having a repo administrator approve
+	// all builds triggered from an outside contributor if this is their first time contributing.
+	ApproveOnce = "first-time"
```

Comment:
A note on this: I am adding this constant as more of a To-Do. I don't have actual functionality for this coded up, but would create a follow-up story to implement this at a later point.